pkg/controller/keepalivedgroup/keepalivedgroup_controller.go

```diff
@@ -408,9 +413,18 @@ func (e *enqueueRequestForReferredKeepAlivedGroup) Update(evt event.UpdateEvent,
 		namespaced, err := getNamespacedName(keepalivedGroup)
 		if err != nil {
 			log.Error(err, "unable to create namespaced name from", "annotation", keepalivedGroupAnnotation, "value", keepalivedGroup)
```

Comment:
this should be a log .Info and add please add to the message something like: `trying with the MetaOld object`
same consideration for the log message below.

Reply:
done